src/encoded/static/components/encyclopedia.js

```diff
@@ -1,49 +1,492 @@
+    };
+
+    // Annotation type facet is not a normal facet, at least one annotation must be selected
+    // Facet filters look like radiobuttons
+    const chooseAnnotationType = (type) => {
+        let newAnnotationType = annotationType;
+        if (annotationType.indexOf(type) > -1 && annotationType.length > 1) {
+            newAnnotationType = annotationType.filter((annotation) => annotation !== type);
+        } else if (annotationType.indexOf(type) === -1) {
+            newAnnotationType = [...annotationType, type];
+        }
+        setAnnotationType(newAnnotationType);
+    };
+
+    // Facet filters are redirected to update the urls that populate the browser
+    const handleFacetClick = (e) => {
+        // If clicked filter belongs to the annotation facet, clear filters, or boolean switch, we do not want to prevent default
+        const isButton = findUpTag(e.target, 'BUTTON');
+        const isClearFilters = e.target.innerText === 'Reset filters';
+        const isBooleanSwitch = findUpClassName(e.target, 'boolean-switch');
+        if (!isButton && !isClearFilters && !isBooleanSwitch) {
+            e.preventDefault();
+            let clickedLink = findUpTag(e.target, 'A');
+            if (!clickedLink && e.target.href) {
+                clickedLink = e.target;
+            }
+            if (clickedLink && clickedLink.href) {
+                const clickedHref = clickedLink.href;
+                getSeriesData(clickedHref, context.fetch).then((response) => {
+                    if (response) {
+                        const newResponse = filterFacet(response, selectedOrganism);
+                        setFacetData(newResponse);
+                        setTotalFiles(newResponse.total);
+                        setBrowserHref(clickedHref);
+                    } else {
+                        setFacetData([]);
+                        setTotalFiles(0);
+                        setVizFilesError(true);
+                    }
+                });
+            }
+        // Reset annotation type and assembly on "Clear filters" click
+        } else if (isClearFilters) {
+            resetPage();
         }
-        return lastRegion;
     };
 
     return (
         <div className="layout">
             <div className="layout__block layout__block--100">
-                <div className="block encyclopedia" data-pos="0,0,0">
-                    <h1>{props.context.title}</h1>
-                    <Panel>
-                        <PanelBody>
-                            <ResultTable
-                                {...props}
-                                searchBase={searchBase}
-                                onChange={context.navigate}
-                                currentRegion={currentRegion}
-                                hideDocType
-                            />
-                        </PanelBody>
-                    </Panel>
+                <div className="block series-search">
```

Comment:
Is it a good idea to follow BEM..? This is me nitpicking but I am just curious.

Reply:
I'm not sure what you mean. How should this be changed?

Reply:
We can talk about this offline if you like.